src/System.Runtime/tests/System/Reflection/MethodBaseTests.cs

```diff
@@ -23,19 +23,34 @@ public static void Test_GetCurrentMethod()
         }
 
         [Fact]
-        public static void Test_GetCurrentMethod_GenericMethod()
+        public static void Test_GetCurrentMethod_GenericMethodDefinition()
         {
             MethodBase m = MyFakeGenericMethod<Byte>();
             
             Assert.Equal("MyFakeGenericMethod", m.Name);
             Assert.Equal("MethodBaseTests", m.ReflectedType.Name);
             Assert.True(m.IsGenericMethod);
             Assert.True(m.IsGenericMethodDefinition);
+            Assert.False(m.IsConstructedGenericMethod);
             Assert.Equal(1, m.GetGenericArguments().Length);
             Assert.Equal("T", m.GetGenericArguments()[0].Name);
         }
 
         [Fact]
+        public static void Test_GetCurrentMethod_GenericConstructedMethod()
```

Comment:
I would name this test "Test_GetCurrentMethod_ConstructedGenericMethod" so the names line up with the api naming.